packages/nodes-base/tsconfig.build.json

```diff
@@ -0,0 +1,14 @@
+{
+	"extends": "./tsconfig.json",
+	"compilerOptions": {
+		"emitDeclarationOnly": true,
+		"declaration": true,
+		"outDir": "./dist/src"
+	},
+	"include": [
+		"src/**/*",
+	],
+	"exclude": [
+		"**/*.spec.ts"
+	]
+}
```

Comment:
1. `declaration: true` is the same as in the parent. Is it necessary here? Same for `exclude` and its contents.
2. `src/**/*` in `include` in the child overrides the parent's. Can we remove it from the parent?
3. The parent is including `nodes/**/*.json` and `credentials/translations/**/*.json` but now `copyAssets` takes care of JSON files. Can we remove these lines from the parent?

Reply:
1. Removed `declaration:true` and `exclude`. Thanks for pointing that out!
2. The parent should still be compiling all the files (for generating all `d.ts` files) and targeting all the files (for IDEs, some IDEs such as Webstorm only take the root tsconfig file into consideration).
3. The JSON files still need to be referenced because otherwise you get an error when importing them throughout `nodes-base`.